app/uk/gov/hmrc/agentclientrelationships/model/invitation/DeleteInvitationRequest.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentclientrelationships.model.invitation
+
+import play.api.libs.json.{Json, OFormat}
+import uk.gov.hmrc.agentclientrelationships.model.invitation.InvitationFailureResponse._
+import uk.gov.hmrc.agentmtdidentifiers.model.ClientIdentifier.ClientId
+import uk.gov.hmrc.agentmtdidentifiers.model.{ClientIdentifier, Service}
+
+import scala.util.Try
+
+case class DeleteInvitationRequest(
```

Comment:
for better readability, because the request body is used to remove an authorisation and not an invitation can we rename this? Maybe something like `RemoveAuthorisationRequest` which matches your new endpoint